packages/interface/src/components/ui/Form.tsx

```diff
@@ -66,6 +66,13 @@ export const ErrorMessage = createComponent("div", tv({ base: "pt-1 text-xs text
 
 export const Textarea = createComponent("textarea", tv({ base: [...inputBase, "w-full"] }));
 
+// eslint-disable-next-line react/display-name
+export const DateInput = forwardRef(({ ...props }: ComponentPropsWithRef<typeof Input>, ref) => (
+  <InputWrapper>
+    <Input ref={ref} {...props} type="date" />
+  </InputWrapper>
+));
```

Comment:
Don't disable eslint rules
```suggestion
export const DateInput = forwardRef((props: ComponentPropsWithRef<typeof Input>, ref) => (
  <InputWrapper>
    <Input ref={ref} {...props} type="date" />
  </InputWrapper>
));
```